packages/client/src/glsp-sprotty/index.ts

```diff
@@ -230,12 +260,12 @@ export {
     CircularNode,
     CircularPort,
     DiamondNode,
-    ForeignObjectElement,
-    HtmlRoot,
-    PreRenderedElement,
+    ForeignObjectElementImpl as ForeignObjectElement,
+    HtmlRootImpl as HtmlRoot,
+    PreRenderedElementImpl as PreRenderedElement,
     RectangularNode,
     RectangularPort,
-    ShapedPreRenderedElement
+    ShapedPreRenderedElementImpl as ShapedPreRenderedElement
 } from 'sprotty/lib/lib/model';
 // ------------------ Library ------------------
 export * from 'sprotty/lib/lib/modules';
```

Comment:
Should we also `export * from 'sprotty/lib/lib/virtualize';`? Seems odd that this is omitted or is it because of the default export?

Reply:
Sure we can add it. It's not part of the original sprotty main index, but I don't see a reason why we should not reexport those symbols as well.